db/migrate/20180609142034_create_users.rb

```diff
@@ -0,0 +1,9 @@
+class CreateUsers < ActiveRecord::Migration[5.1]
+  def change
+    create_table :users do |t|
+      t.string :email, null: false
+      t.string :name
```

Comment:
Is it by expected that you can create user with no name?

Reply:
It will be discussed in the future